docs/zh-CN/docs/sql-manual/sql-functions/string-functions/field.md

```diff
@@ -0,0 +1,63 @@
+---
+{
+    "title": "field",
+    "language": "zh-CN"
+}
+---
+
+<!-- 
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+## field
+### description
+#### Syntax
+
+`field(Expr e, param1, param2, param3,.....)`
+
+在order by子句中，可以使用自定义排序，可以将expr中的数据按照指定的param1，2，3顺序排列。
```

Comment:
![image](https://user-images.githubusercontent.com/10553413/203683448-1f09ddcd-5ae0-486f-8fac-db4c15a811f4.png)

Reply:
Check the logic order by field (name 'banana', 'apple', 'banana') the logic of mysql. return error ?